app/src/main/AndroidManifest.xml

```diff
@@ -10,15 +13,19 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
         <activity
-            android:name=".ChatActivity"
+            android:name=".SplashActivity"
```

Comment:
Splash activity is loosing its trend in market. Rather use home screen as screen with no/less network usage and easy to load content.